gallery/gallery/tool/l10n_cli/l10n_cli.dart

```diff
@@ -70,9 +75,9 @@ Future<void> generateXmlFromArb({
 
     final resourceId = key.substring(1);
     final name = _escapeXml(resourceId);
-    final Map<String, dynamic> metaInfo = bundle[key];
+    final Map<String, dynamic> metaInfo = bundle[key] as Map<String, dynamic>;
```

Comment:
super-nit: you should be able to take out the type specifier if you're manually casting.